File: CrazyBooks.Lib/Models/Relations/Lend.cs
using System;
using CrazyBooks.Lib.Core;

namespace CrazyBooks.Lib.Models
{
    public class Lend : Entity
    {
        public Guid MemberId { get; set; }

        public Member Member { get; set; }

        public Guid BookId { get; set; }

        public Book Book { get; set; }
    }
}

File: CrazyBooks.Web/wwwroot/Scripts/Models/Lend.js
class Lend extends Entity
{
    constructor(json)
    {
        super(json);

        if (json)
        {
            this.Book = json.book;
            this.Member = json.member;
            this.LendedOn = json.lendedOn;
            this.BookComposedName = json.bookComposedName;
        }
        else
        {
            this.LendedOn = new Date();
            this.Book =  null;
            this.Member = null;
        }
    }
}